manifest.json

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Confusing to see the Liquid comment here, lets drop this instead.

Reply:
The frontmatter has to be there because I use jekyll `absolute_url` pipes in the file.